coreff_pappers/pappers.py

```diff
@@ -0,0 +1,226 @@
+import requests
+import json
+import base64
+from odoo.exceptions import UserError
+
+
+def clean_code(code):
+    """Removes any non-numerical character and returns a string"""
+    numbers = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0"]
+    cleaned_code = ""
+
+    for character in code:
+        if character in numbers:
+            cleaned_code += character
+    return cleaned_code
```

Comment:
- using `.isdigit()` is prefered
- best is using assertion instead of input change, if any non numerical is used, throw an exception